integration-commons/src/main/java/org/symphonyoss/integration/exception/IntegrationException.java

```diff
@@ -58,4 +58,14 @@ public IntegrationException(String component, String message, Throwable cause) {
   public IntegrationException(String component, String message, Throwable cause, String... solutions) {
     super(ExceptionMessageFormatter.format(component, message, cause, solutions), cause);
   }
+
+  /**
+   * Constructs a new IntegrationException with the component name and the specified detail message.
```

Comment:
Constructs a new IntegrationException with the component name, the specified detail message, and the solutions provided for this exception.